src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -41,7 +41,7 @@
 diskslist_help = """List of disks to be excluded or included."""
 disk_list_setting_help = """option to decide whether to include or exclude the disk or reset any previous settings to default behavior"""
 target_container_name_help = """The target container to which the DB recovery point should be downloaded as files."""
-infrastructure_encryption_type_help = """Infrastructure encryption must be enabled when configuring encryption of the vault for the first time. Once enabled, infrastructure encryption cannot be disabled. Allowed values: Enabled, Disabled"""
+infrastructure_encryption_type_help = """Use this parameter to Enable/Disable Infrastructure encryption. This must be set when configuring encryption of the vault for the first time. Once Enabled/Disabled, infrastructure encryption setting cannot be changed. Default value: Disabled. Allowed values: Enabled /Disabled"""
```

Comment:
I didn't mean use of /  in Allowed values - there we should use comma. I originally meant in the begining of sentence which is taken care now. Move back to comma at the end